### PR TITLE
[Make.config] Allow '.' on CURRENT_BRANCH_ALPHANUMERIC to match android package names

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

**Review by spouliot — APPROVED** [review]

File: Make.config [hunk 1]
```diff
@@ -57,7 +57,7 @@ CURRENT_BRANCH_SED_ESCAPED:=$(subst |,\|,$(subst &,\&,$(subst $$,\$$,$(subst /,\
 # The branch name in the nuget version has to be alphanumeric, and we follow the semantic versioning spec,
 # which defines "alphanumeric" as the letters, numbers and the dash character (and nothing else).
 # So here we replace all non-alphanumeric characters in the branch name with a dash.
-CURRENT_BRANCH_ALPHANUMERIC:=$(shell export LANG=C; printf "%s" "$(CURRENT_BRANCH)" | tr -c '[a-zA-Z0-9-]' '-')
+CURRENT_BRANCH_ALPHANUMERIC:=$(shell export LANG=C; printf "%s" "$(CURRENT_BRANCH)" | tr -c '[a-zA-Z0-9-.]' '-')
```
Comment (rolfbjarne) [inline Make.config:60]: @dalexsoto why was this necessary? It makes it possible to create package versions that don't conform to the semantic versioning spec (and contradicts the comment just above).

Comment (dalexsoto) [reply Make.config:60]: It was to match the naming of android packages but IIRC it was revealed later
